tests/nix_tests/test_patches.py

```diff
@@ -27,6 +28,8 @@ def test_send_funds_to_distr_mod(evmos_cluster):
     cli = evmos_cluster.cosmos_cli()
     sender = eth_to_bech32(ADDRS["signer1"])
     amt = 1000
+    fee_denom = cli.evm_denom()
```

Comment:
why are we assuming the `fee_denom` to be the `evm_denom`? In case of a 6 dec token it's going to be different right? 🤔

Reply:
with the current implementation in this repository, the evm denom == fee denom